edx_proctoring/tests/utils.py

```diff
@@ -30,10 +30,8 @@ def login_user(self, user):
 
         # Create a fake request to store login details.
         request = HttpRequest()
-        if self.session:
-            request.session = self.session
-        else:
-            request.session = engine.SessionStore()
+
+        request.session = engine.SessionStore()
```

Comment:
Note to reviewers, the change to 1.4.22 broke this helper session manager. 1.4.22 changed the way sessions are managed.